### ISSUE TITLE
redefinition warnings for every Geo::GDAL::FFI->new call

### ISSUE THREAD

**Issue by shawnlaffan** [ajolma/Geo-GDAL-FFI#5]
This is probably a documentation issue. 

The code below results in a slew of subroutine redefinition warnings.  If I change the ```my``` declaration to ```state``` then they disappear.  

If the Geo::GDAL::FFI object is a package singleton then it should probably be clarified in the docs.  

```
use 5.010;
use Geo::GDAL::FFI;

for my $i (0,1) {
    my $g = Geo::GDAL::FFI->new;
}
```

**Comment by ajolma** [comment]
Fixed in 0c46ada.

**Comment by shawnlaffan** [comment]
Thanks Ari.
